tox.ini

```diff
@@ -17,5 +17,8 @@ deps =
     pandas
     scipy
     numba
+    GitPython
+    pytest-mock
```

Comment:
pytest-mock should also be added to `dev` dependencies in `pyproject.toml`, otherwise local test fails